Keas.Core/Services/IEmailService.cs

```diff
@@ -272,20 +198,35 @@ public async Task SendNotificationMessage(User user)
                 _dbContext.Notifications.Update(notification);                
             }
             await _dbContext.SaveChangesAsync();
+        }
 
-            var mimeType = new System.Net.Mime.ContentType("text/html");
-
-            var alternate = AlternateView.CreateAlternateViewFromString(message.Body, mimeType);
-            message.AlternateViews.Add(alternate);

```

Comment:
@jpknoll Does Sparkpost have AlternateViews?

Reply:
You can specify .Text and .Html views on the Transaction.Content.